### ISSUE TITLE
Hi! have you tried implement MVCNN based on VGG 19 not the Alexnet?

### ISSUE THREAD

**Issue by yifanfeng97** [WeiTang114/MVCNN-TensorFlow#12]
I used Tensorflow, and i was besed on VGG19. But I just got the OOM error! I had reused the variable in the loop! I used TITAN X.

**Comment by youkaichao** [comment]
I was also wandering.
In the paper, the author use VGG-M and VGG-VD. So why many people implement it using AlexNet?(the Caffe implementation uses AlexNet as well)

**Comment by WeiTang114** [comment]
@youkaichao  
My reason is I had difficulty finding usable pretrained weight for VGG-M (and can be converted for Tensorflow) when I began this project 😅. 
(VGG-M was implemented in matconvnet)

**Comment by WeiTang114** [comment]
@XDUfyf 
I haven't tried VGG19. Did you try reducing the batch size?

**Comment by yifanfeng97** [comment]
@WeiTang114 I have tried reducing the batch size, and I set batch size with 1. It did not work, too.

At the same time, I met a strange problem. 
when I tried train with single view not multi-view(based on VGG19, my multi-view didn't work. so I 
 temporarily use single view as input ) in ModelNet40. If I set batch size as 64 ,training would get nan in step 404. however I set batch size as 16, training would get nan in step 1619( 64 = 16*4 and 1619 = 404*4). No matter how many times I run. It would get same result. My loss curve is normal.
environment: Tensorflow, Python3, TITAN X , Cuda 8.

![loss](https://user-images.githubusercontent.com/15007108/29506871-4f27b534-8680-11e7-9762-78aa13afb34f.png)

![error 1](https://user-images.githubusercontent.com/15007108/29506923-7ee47eba-8680-11e7-999e-53ed694dd3b5.png)

![error3](https://user-images.githubusercontent.com/15007108/29506881-58bb7fea-8680-11e7-9bee-7f2496063383.png)

**Comment by paulcx** [comment]
The resnet 101 is definitely a good choice to try for better performance.

**Comment by foxfromworld** [comment]
Hi,
Can we train our model from scratch without the pretrained AlexNet model?
Thanks!
